plugins/Monolog/Test/Integration/LogTest.php

```diff
@@ -258,4 +248,27 @@ public static function getLogFileLocation()
     {
         return StaticContainer::get('path.tmp') . '/logs/piwik.test.log';
     }
+
+    public function provideContainerConfig()
+    {
+        return array(
+            'Psr\Log\LoggerInterface' => \DI\get('Monolog\Logger')
+        );
+    }
+
+    private function recreateLogSingleton($backend, $level = 'INFO')
+    {
+        $newEnv = new Environment('test', array(
+            'ini.log.log_writers' => array($backend),
+            'ini.log.log_level' => $level,
+            'ini.log.string_message_format' => self::STRING_MESSAGE_FORMAT,
+            'ini.log.logger_file_path' => self::getLogFileLocation(),
+            'Psr\Log\LoggerInterface' => \DI\get('Monolog\Logger')
```

Comment:
`provideContainerConfig()` is called when the test case creates a new Environment instance, so we'd have to call it here if we put the other lines in that method. But you're right, since we create a new Environment in order to inject the Logger into Log, we don't need the `provideContainerConfig()`, just forgot to remove it.

Note: Will remove in a new commit since PRs depend on this one.